config/main.py

```diff
@@ -464,6 +470,28 @@ def portchannel_member(ctx):
 def add_portchannel_member(ctx, portchannel_name, port_name):
     """Add member to port channel"""
     db = ctx.obj['db']
+    port_channel = db.get_entry('PORTCHANNEL', portchannel_name)
+    if get_interface_naming_mode() == "alias":
+        port_name = interface_alias_to_name(port_name)
+        if port_name is None:
+            ctx.fail("'port_name' is None!")
+    
+    if len(port_channel) == 0:
+        ctx.fail("{} doesn't exists".format(portchannel_name))
+    
+    members = port_channel.get('members', [])
+    if port_name in members:
+        if get_interface_naming_mode() == "alias":
+            port_name = interface_name_to_alias(port_name)
```

Comment:
will this line duplicate with line 475?
if it is 'alias' mode and the port_name is already in members, it will try to convert the name again?

Reply:
Thanks for review comments, I'll fix and run the test cases, will update the PR.